.vscode/tasks.json

```diff
@@ -16,6 +16,36 @@
             "options": {
                 "env": {}
             }
+        },
+        {
+            "label": "Build",
+            "type": "shell",
+            "command": "./build.ps1",
+            "windows": {
+                "command": "./build.ps1"
+            },
+            "group": {
+                "kind": "build",
+                "isDefault": true
```

Comment:
I don't think you're supposed to have multiple default tasks in a group, and this looks like a (better) duplicated of `NDK Build`. I'd suggest removing `NDK Build` and remove `isDefault` from `Copy`.